youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -175,7 +175,7 @@ def _ffmpeg_filename_argument(self, fn):
         # Always use 'file:' because the filename may contain ':' (ffmpeg
         # interprets that as a protocol) or can start with '-' (-- is broken in
         # ffmpeg, see https://ffmpeg.org/trac/ffmpeg/ticket/2127 for details)
-        return 'file:' + fn
+        return 'file:' + fn if fn != '-' else fn
```

Comment:
Something like: `Also leave '-' intact in order not to break streaming to stdout`.